### PR TITLE
Fix Quickstart for FedRAMP

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

## Further comments

```
➜  ~/workspace/mongoCLI git:(CLOUDP-94313) ✗ ./bin/mongocli atlas quickstart
You are creating a new Atlas cluster and enabling access to it.

Press [Enter] to use the default values.

Enter [?] on any option to get help.

[Set up your Atlas cluster]
? Cluster Name [This can't be changed later] Quickstart-1626180399
? Cloud Provider AWS
? Cloud Provider Region US_GOV_EAST_1

We are deploying Quickstart-1626180399...
? Do you want to load sample data into Quickstart-1626180399? Yes

[Set up your database authentication access details]
? Database User Username Quickstart-1626180399
? Database User Password [Press Enter to use an auto-generated password 'SH1pV3NIb3tZ'] SH1pV3NIb3tZ

[Set up your database network access details]
? Access List Entry [Press Enter to use your public IP address '109.115.34.44'] 109.115.34.44

[Connect to Quickstart-1626180399]
? Do you want to access Quickstart-1626180399 with MongoDB Shell? Yes

Creating your cluster... [It's safe to 'Ctrl + C']
-
Loading sample data into your cluster... [It's safe to 'Ctrl + C']
-
Now you can connect to your Atlas cluster with: mongosh -u Quickstart-1626180399 -p SH1pV3NIb3tZ mongodb+srv://quickstart-1626180399.jxkfe.mongodbgov-dev.net

Current Mongosh Log ID: 60ed8e81f3c64601168050ff
Connecting to:    mongodb+srv://quickstart-1626180399.jxkfe.mongodbgov-dev.net/
Using MongoDB:      4.4.5-1-fips-openssl-1.0.2
Using Mongosh Beta: 0.8.2

For mongosh info see: https://docs.mongodb.com/mongodb-shell/

atlas-13pzho-shard-0 [primary]>
```
Screenshot 2021-07-13 at 15 02 52

### CODE REVIEW

File: internal/cli/atlas/atlas.go [hunk 1]
```diff
@@ -49,8 +48,6 @@ func Builder() *cobra.Command {
 		Use:   Use,
 		Short: "Atlas operations.",
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
-			config.SetService(config.CloudService)
```
Comment (gssbzn) [inline internal/cli/atlas/atlas.go:52]: can we condition this? if not set set it to cloud?

Comment (andreaangiolillo) [reply internal/cli/atlas/atlas.go:52]: sure

**Review by gssbzn — APPROVED** [review]
LGTM

**Comment by andreaangiolillo** [comment]
This just missed the 1.19 release :(

**Pull request merged**